flags.go

```diff
@@ -341,6 +344,82 @@ type flagStorage struct {
 	DebugMutex      bool
 }
 
+const PARENT_PROCESS_EXECUTION_DIR_KEY = "MD5Hash38cda02df9fcf2b1bba019000cce0eb6"
```

Comment:
keep the name simple 
GCSFUSE_PARENT_PROCESS_DIR = "gcsfuse-parent-process-dir"

Reply:
done.